### DISCUSSION TITLE
Chrome Dark Mode WC QR Codes

### DISCUSSION THREAD

**Discussion by john-ostrom** [WalletConnect/walletconnect-monorepo#3926] [Developer Support] [answered]
I notice when Wallet Connect presents its QR code, in the Chrome browser with dark mode forced on ( chrome://flags/#enable-force-dark ),  the eye frame and eye ball shapes in the 3 corners are filled. Thus, iOS cannot scan in the presented QR code. Some other QR scanners can handle this incorrect QR code. Is there a potential work around for this QR problem  with Wallet Connect? For instance, the Etherscan website has a web 3 login for wc:

![qrscreenshot](https://github.com/WalletConnect/walletconnect-monorepo/assets/99298852/f18ed3c2-896f-40e2-8cbf-d31b2f8ef422)

**Comment by glitch-txs** [comment, marked as answer]
hi, you can hardcode the theme config. we've changed the modal UI in v3

**Reply by john-ostrom** [reply]
I'm running it on http://localhost:3000/ . It appears fixed. I can see the eye frame and balls as expected when forcing dark mode in Chrome.

Though, I figure sites will need to upgrade in the long run.
